bridge/client/app/_components/ktb-root-events-list/ktb-root-events-list.component.scss

```diff
@@ -1,26 +1,41 @@
 @import '../../../variables';
 @import '~@dynatrace/barista-components/core/src/style/variables';
 
-.dt-tag.error {
-  border-color: $error-color;
-}
-.dt-tag.warning {
-  border-color: $orange-400;
-}
+.ktb-root-events-list {
 
-.event-icon {
-  width: 20px;
-  height: 20px;
-}
+  .dt-tag.error {
+    background-color: $error-color;
+    color: $white;
+  }
+  .dt-tag.warning {
+    border-color: $orange-400;
+  }
 
-.event-icon.error {
-  fill: $error-color;
-}
+  .event-icon {
+    width: 20px;
+    height: 20px;
+  }
 
-.event-icon.warning {
-  fill: $orange-400;
-}
+  .event-icon.error {
+    fill: $error-color;
+  }
+
+  .event-icon.warning {
+    fill: $orange-400;
+  }
+
+  .event-icon.success {
+    fill: $cta-color-active;
+  }
+
+  .ktb-selectable-tile.ktb-tile-error .ktb-selectable-tile-container {
+    border: 1px solid $gray-200;
```

Comment:
is this purposely `$gray-200` instead of `$gray-300` like in line 38? if they should have the same color you could use the same style for all three

Reply:
That is on purpose. Since I want only the left border to be red. By default the `.ktb-selectable-tile.ktb-tile-error` has a red border all around.